kasten/k10-minimum-retention/k10-override-minimum-retentions.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: k10-policy-minimum-retention
+  annotations:
+    policies.kyverno.io/title: Minimum Backup Retention
+    policies.kyverno.io/category: Kasten K10
+    policies.kyverno.io/subject: Policy
+    policies.kyverno.io/description: >-
+      K10 Policy resources can be validated to adhere to common compliance retention standards. 
+      Uncomment the number of copies you want to maintain according to GFS retention.
+      Check below for a reference of common compliance requirements:
+      Federal Information Security Management Act (FISMA): 3 Years 
+      National Energy Commission (NERC): 3 to 6 Years  
+      Basel II Capital Accord: 3 to 7 Years 
+      Sarbanes-Oxley Act of 2002 (SOX): 7 Years 
+      Health Insurance Portability and Accountability Act (HIPAA):  6 Years   
+      National Industrial Security Program Operating Manual (NISPOM): 6 to 12 Months 
+      Payment Card Industry Data Security Standard (PCI-DSS): Varies
+      This policy can also be used go reduce retentions to enforce cost optimization.
+spec:
+  rules:
+  - name: k10-policy-minimum-retention
+    match:
+      any:
+      - resources:
+          kinds:
+          - config.kio.kasten.io/v1alpha1/Policy
+    mutate: 
+      # HIPAA example (replace existing values only: ie yearly with 6)
```

Comment:
Comments are nice to show users an alternate policy configuration for different use cases, but you may want to give this in a JSON patch form rather than a strategic patch form, which as you know the Kasten Policy resource does not support.